README.md

```diff
@@ -111,7 +75,7 @@ The figure below show the architecture of TiSpark.
 
 ![architecture](./docs/architecture.png)
 
-+ TiSpark integrates well with the Spark Catalyst Engine. It provides precise control of computing, which allows Spark to read data from TiKV efficiently. It also supports index seek, which significantly improves the performance of the point query execution.
++ TiSpark integrates well with the Spark Catalyst Engine. It provides precise control of comfputing, which allows Spark to read data from TiKV efficiently. It also supports index seek, which significantly improves the performance of the point query execution.
```

Comment:
? seems not right

Reply:
coopy from matser, i think both are wrong